sig-multicluster/namespace-sameness-position-statement.md

```diff
@@ -92,10 +96,29 @@ thus namespce sameness applies), the implementation of multi-cluster services
 probably should not be on-by-default.  The details of how multi-cluster
 services will work are an area for innovation, but ideas include:
   * Opt-in: services must be "exported" to be merged across clusters
-  * Opt-out: services or namespace can be opted out of service merging
+  * Opt-out: services or namespaces can be opted out of sameness
   * Different discovery: merged services and "raw" services use different names
     or other discovery mechanisms
```

Comment:
>  but some could be reserved/un-exported, or perhaps more confusingly, exported but "not-same"?

I understand namespace sameness to allow for only the first, not the second (I would prefer the line here should say what is being opted out of is, as originally, "service merging", or even better "exporting", but definitely not as proposed here "sameness"). Most (all?) MCS implementations today are opt-in so implementation wise opt-out is only its corollary: by not having ServiceExports made in 1-N clusters with the same named namespace/service in the first place or removing them. Istio et al probably have more to say on opt out patterns of behavior since those implementations are more on-by-default.

Reply:
I'll update.  At the time this was written, MCS semantics had not really been pinned down.  That said, I like it as an example, but I don't want to inline that spec here.

Will try to balance.